code/modules/mob/mob_helpers.dm

```diff
@@ -491,3 +491,6 @@ mob/proc/get_standard_bodytemperature()
 			m_intent = MOVE_INTENT_RUN
 
 	client.screen[screen_num].update_icon(src)
+
+/mob/proc/can_hear()
+	. = !isdeaf(src)
```

Comment:
This is dealt with differently in TG but we don't have ears currently. We can change this later when we do.